ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_search.py

```diff
@@ -1,3 +1,35 @@
 # global
+from hypothesis import strategies as st
 
 # local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+@handle_frontend_test(
+    fn_tree="paddle.argmax",
+    dtype_x_and_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_num_dims=1,
+        valid_axis=True,
+        force_int_axis=True,
+    ),
+    keepdim=st.booleans(),
+)
+def test_paddle_argmax(
+    dtype_x_and_axis,
+    keepdim,
+    frontend,
+    test_flags,
+    fn_tree,
+):
+    input_dtypes, x, axis = dtype_x_and_axis
+    helpers.test_frontend_function(
```

Comment:
Please test for `dtype` as well :pleading_face:

Reply:
I've skipped `dtype` test and added a comment. Thank you 😃

Reply:
Sorry one last nitpick, by below the test I meant somewhere right below the function definition line :sweat_smile: Sorry should've worded that better

Reply:
Sorry, I misunderstood. Would this be better?